packages/starlight/schema.ts

```diff
@@ -165,5 +165,8 @@ export function docsSchema() {
 					content: z.string(),
 				})
 				.optional(),
+
+			/** Pagefind indexing for this page - set to false to disable. */
+			pagefind: z.boolean().optional().default(true),
```

Comment:
`.default()` here implies `.optional()`, so we can simplify that:

```suggestion
			pagefind: z.boolean().default(true),
```